app/pods/publications/publication/translations/documents/route.js

```diff
@@ -10,7 +10,7 @@ export default class PublicationsPublicationTranslationsDocumentRoute extends Ro
     const queryProperties = {
       include: [
         'file',
-        'publication-subcase'
+        'publication-subcase-source-for'
```

Comment:
Deze include gaat vooral om de referentie documenten denk ik.
Dus die vorige was wel juist ?

Reply:
Dit is juist. Ik heb deze relatie moeten hernoemen gezien een Piece nu aan een PublicationSubcase kan gelinkt zijn als sourceDocument of als correctionDocument. Ik was het hier vergeten te wijzigen.

referenceDocuments is een property is van PublicationFlow. Het is hier volgens mij niet de bedoeling deze te tonen.

Reply:
Ah juist ja, had niet goed gekeken.
ref docs zijn gelinkt aan pub-flow, hier gaat het over de link tussen piece en publicatie subcase voor "voor drukproef"